README.md

```diff
@@ -553,6 +553,7 @@ A third-party Golang SDK for Jackett is available from [webtor-io/go-jackett](ht
  * Torrentland
  * TorrentLeech (TL)
  * TorrentLeech.pl
+ * TorrentMasters
```

Comment:
Since registrations are currently invite-only please add this to the readme. Thanks.
```suggestion
 * TorrentMasters [![(invite needed)][inviteneeded]](#)
```